### ISSUE TITLE
[MSFT-111] Member registration page: The 'email tips', 'user name tips' and 'password tips' are designated with level 3 headings instead of level 2.

### ISSUE THREAD

**Issue by veshu** [importtest/test#95]
**Steps to Reproduce**
1. Go to the Members registration page:https://accounts.topcoder.com/member/registration. 2. Ensure that the screen reader is switched on. 3. Press tab till your arrive at the 'enter your email id' text box. 4. Remove your focus from the text box and press the down arrow key. 5. Screen reader will announce 'email tips heading level 3' followed with a few tips to create a valid mail id. 6. Do the same for the 'password' and the 'user name' fields. 7. Notice that the 'email tips' 'password tips' and 'user name tips' are being announced as level 3 headings.

**Expected Result**
These should be level 2 headings since 'join topcoder' is level 1 heading.

**Actual Result**
email tips' 'password tips' and 'user name tips' are being announced as level 3 headings instead of level 2.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2016
